jobs/release.yaml

```diff
@@ -54,9 +54,9 @@
           type: user-defined
           name: snap_version
           values:
-            - 1.20/edge
-            - 1.19/edge
-            - 1.18/edge
+            - 1.21/stable
+            - 1.20/stable
+            - 1.19/stable
```

Comment:
I moved these to edge [here](https://github.com/charmed-kubernetes/jenkins/pull/691) because I noticed during the last bugfix release that we were about to release cdk-addons changes completely untested. We can't revert this back to stable staps unless we have cdk-addons test coverage somewhere else in the bugfix release process.

Reply:
After discussing our need to test with a pre-stable `cdk-addons`, and based on the fact that `cdk-addons` publishes edge through candidate on each build. i've moved these to `1.xx/candidate`.  Hopefully it's more clear that `validate-charm-bugfix` is going to use candidate charms with candidate snaps now.